packages/theme/src/theme.ts

```diff
@@ -104,7 +104,7 @@ export enum ThemeVariant {
   AtB,
   Nfk,
   FRAM,
-  TFK,
+  TROMS,
```

Comment:
Should these be all capitalised? "AtB" and "Nfk" are not.

Reply:
I didn't notice that. I can update and use `Troms` instead